prow/git/git.go

```diff
@@ -341,16 +348,7 @@ func retryCmd(l *logrus.Entry, dir, cmd string, arg ...string) ([]byte, error) {
 
 func (r *Repo) Diff(head, sha string) (changes []string, err error) {
```

Comment:
Nit: Maybe mention in the godoc that `ThreeDotDiff` also exists?

Reply:
Thanks for the review. Updated the PR.